src/chains.ts

```diff
@@ -1195,6 +1195,44 @@ export const OASIS_TESTNET: Chain = {
   blockExplorerUrls: ['https://testnet.explorer.emerald.oasis.dev'],
 }
 
+export const GODWOKEN_MAINNET: Chain = {
+  id: 'godwoken_mainnet',
+  chain_id: 71394,
+  name: 'Godwoken Testnet',
+  symbol: 'CKB',
+  mainnet: true,
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/godwoken.png',
+  pangolin_is_live: false,
+  tracked_by_debank: false,
+  supported_by_gelato: false,
+  rpc_uri: 'https://mainnet.godwoken.io/rpc',
+  nativeCurrency:{
+    name: 'Nervos Network',
+    symbol: 'CKB',
+    decimals: 18,
+  },
+  blockExplorerUrls: ['https://layerview.io'],
+}
+
+export const GODWOKEN_TESTNET: Chain = {
+  id: 'godwoken_mainnet',
+  chain_id: 71393,
+  name: 'Godwoken Testnet',
+  symbol: 'CKB',
+  mainnet: false,
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/godwoken.png',
+  pangolin_is_live: false,
+  tracked_by_debank: false,
+  supported_by_gelato: false,
+  rpc_uri: 'https://godwoken-testnet-web3-rpc.ckbapp.dev',
```

Comment:
Not sure this is correct. Need to confirm it's not x/rpc/eth-wallet like Godwoken is

Reply:
_L2 Godwoken Testnet explorer: https://v1.aggron.gwscan.com/_ seems to be this link.

Reply:
https://startwithnervos.com/godwoken

Reply:
https://godwoken-testnet-web3-v1-rpc.ckbapp.dev/ it seems to be this link from what I see here : https://startwithnervos.com/godwoken